### DISCUSSION TITLE
Show approach circle on Target mode when necessary

### DISCUSSION THREAD

**Discussion by Bryceed** [ppy/osu#14419] [Ideas]
Depending on the map (and the song), the beatmap doesn't always stay at the same beat until the end of the song, causing the (obvious) fail when click into the circle out of the new timing, which is difficult to know when this happens, the player's intuition is just to keep pressing in the rhythm of the background beat (with metronome).

The approximation circle should be displayed to the user in these cases, as it happens in the first note of the game, since the purpose of the mode is that you worry more about the cursor than the keys, just respecting the "4-beats steps " of the song as the notes get farther and farther apart.

https://user-images.githubusercontent.com/42657376/130284597-53ae3a2b-6f43-4193-af7c-d3f00fe4ac44.mp4

If the person doesn't know the map with these BPM inconsistencies and plays this mode, obviously it will fail. 

Two approaches circles would be ideal to show to the user (visually) the new rhythm that song has taken, since as the player would fail if they continued at the previous rhythm.

**Comment by peppy** [comment]
This is specific to the lazer realisation of target mod (an issue the one of cutting edge does not encounter).

I agree this is pretty weird and hard to follow, especially when this is a mod more about positional accuracy/practice rather than timing. We may want to consider avoiding these changes in timing rather than marking them.
